src/components/OpenInIgvDialog/index.tsx

```diff
@@ -183,15 +225,71 @@ const constructGDSLocalIgvUrl = async (props: { bucketOrVolume: string; pathOrKe
 
   const idx = encodeURIComponent(idxFilePresignUrl);
   const enf = encodeURIComponent(filePresignUrl);
-  const name = pathOrKey.split('/').pop() ?? pathOrKey;
-  return `http://localhost:60151/load?index=${idx}&file=${enf}&name=${name}`;
+
+  return `http://localhost:60151/load?index=${idx}&file=${enf}&name=${igvName}`;
 };
 
-const constructS3LocalIgvUrl = async (props: { bucketOrVolume: string; pathOrKey: string }) => {
-  const { bucketOrVolume, pathOrKey } = props;
+const constructS3LocalIgvUrl = (props: {
+  igvName: string;
+  bucketOrVolume: string;
+  pathOrKey: string;
+}) => {
+  const { bucketOrVolume, pathOrKey, igvName } = props;
 
-  const name = pathOrKey.split('/').pop() ?? pathOrKey;
   const file = `s3://${bucketOrVolume + '/' + pathOrKey}`;
 
-  return `http://localhost:60151/load?file=${encodeURIComponent(file)}&name=${name}`;
+  return `http://localhost:60151/load?file=${encodeURIComponent(file)}&name=${igvName}`;
+};
+
+/**
+ *
+ * We wanted to show more info in the name parameter when opening in IGV
+ * Ref: https://umccr.slack.com/archives/CP356DDCH/p1707116441928299?thread_ts=1706583808.733149&cid=CP356DDCH
+ *
+ * The desired outcome is to include libraryId, sampleId, type, and filetype
+ * Desired output: SBJ00000_WGS_L0000000_PRJ00000_tumor.bam
```

Comment:
Oh, sorry Will. Spec change a bit. Could you remove sample `type` field, pls?

It should just be the `SubjectID_LibraryID_` prepending only.

e.g. `SBJ00000_L0000000_PRJ00000_tumor.bam`

Reply:
Sorry just thought I could go for an extra mile here to include the type. But, yes could follow what we agree on slack. Changing it ...